scripts/jenkins/ardana/ansible/roles/input_model_generator/templates/mkcloud.config.j2

```diff
@@ -39,7 +39,7 @@ export libvirt_type=physical
 # export networkingmode=
 # export nosetestparameters=
 # export tempestoptions=
-# export cephvolumenumber=
+export cephvolumenumber=1
```

Comment:
Isn't this for the internal ceph cluster? Are we going to support some older cloud versions on this new platform?

Reply:
Believe it or not, this is also required for swift to be installed (pasting from qa_crowbarsetup.sh:

```
    # C2: ceph or swift is only possible with at least one volume
    if [[ $cephvolumenumber -lt 1 ]] ; then
        deployswift=
        deployceph=
    fi
```

Reply:
ah, right swift (ab)uses some of ceph code in mkcloud/qa_crowbarsetup.